README.md

```diff
@@ -317,6 +317,16 @@ end
 
 ```
 
+In production your compiled JS is cached by each JS VM. In development this cache is cleared whenever you change a `*.jsx*` file in `app/assets/javascripts`. In order to clear this cache when you edit files that do not fit this pattern, you need to tell Rails which paths to watch for changes.
+
+```ruby
+# config/environments/application.rb
+# These are the defaults if you dont specify any yourself
+MyApp::Application.configure do
+  config.watchable_files.concat Dir["#{app.root}/app/assets/javascripts/**/*.jsx*"]
```

Comment:
Good point, fixed! Interestingly enough this approach seemed to work for me because I used Dir["#{Rails.root}/app/assets/javascripts/react/**/*"]), which contained the directories too.